main.go

```diff
@@ -11,7 +11,12 @@ import (
 func main() {
 	var err error
 
-	err = LoadDatabase("sqlite:file=sqlite3.db")
+	fp := os.Getenv("COMMENTO_DATABASE_FILE")
+	if fp == "" {
+		fp = "sqlite3.db"
+	}
+
+	err = LoadDatabase("sqlite:file=" + fp)
```

Comment:
Directly do `os.Getenv("...")` after you've made the config.go change

Reply:
Done as well